extensions/notebook/src/types.d.ts

```diff
@@ -3,13 +3,45 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { JupyterController } from './jupyter/jupyterController';
-
 /**
  * The API provided by this extension.
  *
  * @export
  */
 export interface IExtensionApi {
-	getJupyterController(): JupyterController;
+	getJupyterController(): IJupyterController;
+	registerPackageManager(providerId: string, packageManagerProvider: IPackageManageProvider): void
```

Comment:
this may be a silly question but i'll ask it anyway... do you have other code in-flight that leverages these new api's? do you anticipate that this set will drastically change in the near future?

Reply:
I wanted to send the extension that's using in in different PR

---
In reply to: [350548767](https://github.com/microsoft/azuredatastudio/pull/8473#discussion_r350548767) [](ancestors = 350548767)